api/api.go

```diff
@@ -202,7 +201,6 @@ func (api *API) setupRoutes() {
 	ipfsPrivateProtected.GET("/network/:name", api.getIPFSPrivateNetworkByName)                          // admin locked
 	ipfsPrivateProtected.POST("/ipfs/check-for-pin/:hash", api.checkLocalNodeForPinForHostedIPFSNetwork) // admin locked
 	ipfsPrivateProtected.POST("/ipfs/object-stat/:key", api.getObjectStatForIpfsForHostedIPFSNetwork)
-	ipfsPrivateProtected.POST("/ipfs/object/size/:key", api.getFileSizeInBytesForObjectForHostedIPFSNetwork)
```

Comment:
just a thought but havent really addressed it, shouldn't we have these nested paths (`/ipfs/`) under sub-routers?

Reply:
good call, this is something we should definitely address before our next release :ok_hand: